apps/etterlatte-utbetaling/src/test/kotlin/utbetaling/iverksetting/oppdrag/OppdragMapperTest.kt

```diff
@@ -1,15 +1,21 @@
 package no.nav.etterlatte.utbetaling.iverksetting.oppdrag
 
+import no.nav.etterlatte.utbetaling.iverksetting.utbetaling.Saktype
 import no.nav.etterlatte.utbetaling.utbetaling
+import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Assertions.assertNotNull
 import org.junit.jupiter.api.Test
 
 internal class OppdragMapperTest {
 
     @Test
     fun oppdragFraVedtak() {
-        val oppdrag = OppdragMapper.oppdragFraUtbetaling(utbetaling(), true)
-
-        assertNotNull(oppdrag)
+        val oppdrag = OppdragMapper.oppdragFraUtbetaling(utbetaling(sakType = Saktype.BARNEPENSJON), true)
+        assertEquals(oppdrag.oppdrag110.kodeFagomraade, "BARNEPE")
+    }
+    @Test
+    fun oppdragFraVedtakOMS() {
+        val oppdrag = OppdragMapper.oppdragFraUtbetaling(utbetaling(sakType = Saktype.OMSTILLINGSSTOENAD), true)
+        assertEquals(oppdrag.oppdrag110.kodeFagomraade, "OMSTILL")
```

Comment:
kan godt sjekke første oppdragslinje her også for å verifisere at `kodeKlassifik` har blitt riktig på denne, og kanskje tilsvarende på testen over for barnepensjon.

Reply:
gjort